spec/controllers/presentations_controller_spec.rb

```diff
@@ -90,6 +90,17 @@
         end
       end
     end
+
+    describe 'DELETE #destroy' do
+      let!(:presentation) { FactoryGirl.create(:presentation) }
```

Comment:
doesnt need to be a `let!` you call this method in the subject so the presentation will get initialised then